project/Game.py

```diff
@@ -171,21 +169,21 @@ def setUp(self):
         self.game.started = False
 
     def test_add_landmark(self):
-        self.assertTrue(self.game.add_landmark("New York", "Gift given by the French", "statue of liberty"),
-                        "Failed to add landmark")
+        self.assertTrue(self.game.add_landmark("New York", "Gift given by the French", "statue of liberty")
+                        , "Failed to add landmark")
```

Comment:
comma should be on the line above, gives a clear indication that the line doesn't end there